inference-engine/thirdparty/clDNN/src/layout_optimizer.cpp

```diff
@@ -1107,6 +1113,17 @@ bool layout_optimizer::are_data_types_suitable_for_onednn(program_node& node) {
     return false;
 }
 
+bool layout_optimizer::are_layouts_suitable_for_onednn(program_node& node) {
```

Comment:
What about `is_data_padded`, instead? Then `true` and `false` need to be swapped.